src/Render.DocPage.fs

```diff
@@ -9,17 +9,31 @@ open Fable.Import
 
 let private renderPage (menu : React.ReactElement option) (tocContent : string) (pageContent : string) =
     Columns.columns [ Columns.IsGapless ]
-        [ Column.column [ Column.Width (Screen.Desktop, Column.Is2)
-                          Column.Width (Screen.Touch, Column.Is3)
+        [ // TODO: Make this nicer
+          Button.button [ Button.Modifiers [ Modifier.IsHidden (Screen.Tablet, true) ]
+                          Button.Props [ Id "toc-toggle"
+                                         Style [ Position "fixed"
+                                                 Bottom "0"
+                                                 Right "0"
+                                                 Margin "10px"
+                                                 ZIndex "1000" ] ] ]
+                        [ str "Menu" ]
+
+          Column.column [ Column.Width (Screen.Desktop, Column.Is2)
+                          Column.Width (Screen.Tablet, Column.Is3)
+                          Column.Width (Screen.Mobile, Column.IsFull)
                           Column.CustomClass "full-height-scrollable-content"
```

Comment:
Here is the place where we can add `is-hidden-mobile` via `Modifier.IsHidden (Screen.Mobile, true)`